build.gradle

```diff
@@ -7,6 +7,9 @@ plugins {
 repositories {
     mavenLocal()
     mavenCentral()
+    maven {
+        url "http://repository.rocana.com/content/groups/com.scalingdata"
```

Comment:
We don't have a `com.rocana` in our nexus, so artifacts are still published here.